ci/ci

```diff
@@ -0,0 +1,367 @@
+        ap.add_argument('--configure', const='called', nargs='?',
+                        help='configure pbs, defaults to \'--prefix=/opt/pbs --enable-ptl\' \n eg --configure="--enable-ptl" ')
+        ap.add_argument('--make', help='run make command in pbs',
+                        const='called', nargs='?')
+        ap.add_argument(
+            '--install', help='run install command in pbs', const='called', nargs='?')
+        ap.add_argument('--test', const='called', nargs='?',
+                        help='run ptl tests on pbs\n --test default will run \'smoke tag\' for ptl \n eg --test="-f functional/pbs_qmgr.py" ')
+        ap.add_argument('--pbs-stop', const='called',
+                        help='stops all pbs processes inside container including ptl', nargs='?')
+        ap.add_argument('--run', const='called',
+                        help='runs complete install and test for pbs \n eg --run="-f functional/pbs_qmgr.py"', nargs='?')
+        ap.add_argument('--start', const='called',
+                        help='start container, optionally add platform name \n ex --start="ubuntu" ', nargs='?')
+        ap.add_argument('--build', const='called',
+                        help='forces the container to build dependencies, use as a suboption to --start', nargs='?')
+        ap.add_argument('--delete', const='called',
+                        help='destroy pbs container', nargs='?')
+        ap.add_argument('--local', const='called',
+                        help='Simply run the travis script locally, without spawning any containers', nargs='?')
+        ap.add_argument('--pbs-dir', const='called',
+                        help='provide directory location for pbs for which you want ci to run', nargs='?')
+
+        if len(sys.argv) < 2:
+            log_error('\nToo few arguments\n')
+            ap.print_help()
+            sys.exit(1)
+        args = ap.parse_args(sys.argv[1:])
+    except Exception as e:
+        log_error(e)
+    if args.pbs_dir != None:
+        pbs_dirname = os.path.join(args.pbs_dir, 'ci')
+        try:
+            os.chdir(pbs_dirname)
+        except Exception as e:
+            log_error(e)
+            log_error("Was correct pbs path provided")
+            sys.exit(1)
+    if args.delete == None and args.start == None and args.local == None:
+        ret = check_prerequisites()
+        if ret == 1:
+            log_error(
+                "container build failed, build logs can be found in " + build_log_path)
+            sys.exit(1)
+
+    try:
+        if args.configure != None:
+            val = args.configure
+            configure_opt = os.path.join(pbs_dirname, '.configure_opt')
+            if val == 'called':
+                val = read_from_file(configure_opt)
+                if val == '':
+                    val = '--prefix=/opt/pbs --enable-ptl'
+            log_info("using configure parameters : " + val)
+            write_to_file(configure_opt, val)
```

Comment:
we need not call write_to_file() if read_from_file() returned a non-empty string.